### PR TITLE
feature: adding more flexibility to get redis metrics

### PR DESCRIPTION
**Pull request by deimosfr**
Hi,

To follow this pull request #15, I've implemented a new feature to be able to configure metrics directly from the configuration file which kind of metrics you want to grab and which kind of metrics it is
(gauge, counter...)

Hope this will be merged.

Thanks

### CODE REVIEW

File: redis.conf [hunk 1]
```diff
@@ -14,5 +14,26 @@
     # Un-comment to use AUTH
     #Auth "1234"
     Verbose false
+    # Catch Redis metrics (prefix with Redis_)
+    Instance "redis-python_custom"
```
Comment (powdahound) [inline redis.conf]: Not sure a default `Instance` is needed here in this single-instance case.

**Comment by powdahound** [comment]
This looks like a great improvement. Thanks! Have left a few comments.

File: redis_info.py [hunk 2]
```diff
@@ -15,6 +15,8 @@
 #
 # Authors:
 #   Garret Heaton <powdahound at gmail.com>
+# Contributors:
+#   Pierre Mavro <p.mavro@criteo.com> / Deimosfr <deimos at deimos.fr>
```
Comment (powdahound) [inline redis_info.py]: Would be nice to link to https://github.com/powdahound/redis-collectd-plugin/graphs/contributors here too as there have been other contributors too

**Comment by deimosfr** [comment]
Modifications done

**Comment by powdahound** [comment]
Thanks! I don't have an active redis or collectd instance to test this stuff on at the moment. Have you been running this in a real environment successfully? If so, happy to merge on good faith.

**Comment by deimosfr** [comment]
Works fine on a test server. Waiting for your merge request to deploy on 34 instances ;-)

**Comment by powdahound** [comment]
Deploy away! :)

**Comment by ivanyu** [comment]
Hello.
Seem this PR breaks metrics like `db0-keys`.
Or probably I have failed to add it to the config correctly?

**Pull request merged**